pkg/apis/psmdb/v1/perconaservermongodbbackup_types.go

```diff
@@ -13,10 +13,11 @@ import (
 type PerconaServerMongoDBBackupSpec struct {
 	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
 	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
-	PSMDBCluster string              `json:"psmdbCluster,omitempty"` // TODO: Remove after v1.15
-	ClusterName  string              `json:"clusterName,omitempty"`
-	StorageName  string              `json:"storageName,omitempty"`
-	Compression  pbm.CompressionType `json:"compressionType,omitempty"`
+	PSMDBCluster     string              `json:"psmdbCluster,omitempty"` // TODO: Remove after v1.15
+	ClusterName      string              `json:"clusterName,omitempty"`
+	StorageName      string              `json:"storageName,omitempty"`
+	Compression      pbm.CompressionType `json:"compressionType,omitempty"`
+	CompressionLevel *int64              `json:"compressionLevel,omitempty"`
```

Comment:
Why `int64` and not simply `int`? PBM uses `int` in any case.

Reply:
`int` has no fixed size. it could be 32 or 64 bit. I'd prefer to make size consistent and not depend on a running system.
but I don't see strict requirements to not use `int` here